plib/scripts/pre-install.php

```diff
@@ -3,7 +3,7 @@
 
 if (pm_ProductInfo::isWindows()) {
     if (!file_exists(PRODUCT_ROOT . '\dns\bin\rndc.exe')) {
-        echo "BIND DNS Server is not installed.";
+        echo "BIND DNS Server is not installed. Please download it to `" . PRODUCT_ROOT . 'dns\bin\rndc.exe' . "`.";
```

Comment:
To install Slave DNS Manager in Plesk for Windows, copy the "rndc.exe" file to the <folder_path> folder.